cmd/osbuild-image-tests/main_test.go

```diff
@@ -309,6 +315,10 @@ func testImage(t *testing.T, testcase testcaseStruct, imagePath, outputID string
 	}
 
 	if testcase.Boot != nil {
+		if common.CurrentArch() == "aarch64" && !kvmAvailable() {
```

Comment:
I think we could/should skip the tests if `kvm` (or equivalent) is not available, independent of the architecture, no? (This could be in a follow-up if this is not obvious).

Reply:
I think I would also move this condition inside the `boot` sub-test and use `t.Skip`, this way the output log would be more consistent I think.

Reply:
I wanted to avoid breaking other architectures tests so I'd prefer a follow up.

Reply:
Fair enough!